mu/logic.py

```diff
@@ -1303,7 +1305,9 @@ def check_for_shadow_module(self, path):
         return False.
         """
         logger.info('Checking path "{}" for shadow module.'.format(path))
-        filename = os.path.basename(path).replace(".py", "")
+        filename = (
+            os.path.basename(path).replace(".pyw", "").replace(".py", "")
```

Comment:
Maybe we should actually do a case-insensitive replace here? Not sure it's worth the extra work though.

Reply:
Makes sense, we accept uppercase extensions so IMO we should do case-insensitive replace now rather than wait for a user to hit this corner case. Trying it.